app/models/registration.rb

```diff
@@ -531,6 +534,34 @@ def serializable_hash(options = nil)
     super(DEFAULT_SERIALIZE_OPTIONS.merge(options || {}))
   end
 
+  def self.bulk_auto_accept(competition)
+    if competition.waiting_list.present?
+      competition.waiting_list.entries.each do |r_id|
+        Registration.find(r_id).attempt_auto_accept
+        # It isnt clear to me why this is needed - but without it, the auto-accepted items getting removed from the waiting list reappear
+        # on the waiting list in the test body, even when calling waiting_list.reload there
+        # I have also tried assigning competition.waiting_list.entries to a variable and iterating over that, but it didnt help
+        competition.waiting_list.reload
```

Comment:
Can you point me to a specific test case that should work but doesn't without this line? That would make it easier for me to debug

Reply:
`spec/requests/api_registrations_spec.rb:1379` (PATCH #bulk_auto_accept::triggers bulk auto accept via API route)